bundles/CoreBundle/Migrations/Version20200820151104.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+
+class Version20200820151104 extends AbstractPimcoreMigration
+{
+    public function doesSqlMigrations(): bool
+    {
+        return true;
+    }
+
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        try {
+            $this->addSql('ALTER TABLE `tags` CHANGE COLUMN `name` `name` varchar(255) DEFAULT NULL COLLATE utf8_bin AFTER `idPath`');
+            $this->addSql('ALTER TABLE `tags` ADD UNIQUE INDEX `idPath_name` (`idPath`,`name`)');
```

Comment:
Some sort of migration of duplicate tags must happen here, right?

Reply:
@dvesh3 I agree with @NiklasBr - we should rename duplicate tags in the migration to ensure seamless upgrades.

Reply:
Added migration for renaming duplicate tags with console warnings.